opencti-platform/opencti-front/src/private/components/threats/intrusion_sets/IntrusionSetCard.tsx

```diff
@@ -31,6 +31,18 @@ const IntrusionSetCardFragment = graphql`
             }
           }
         }
+        images: importFiles(prefixMimeType: "image/") {
+          edges {
+            node {
+              id
+              name
+              metaData {
+                inCarousel
+                  description
+              }
+            }
+          }
+        }
```

Comment:
I have a performance issue with this part on intrusion set list (and probably on Threat actors groups and individuals), since it's loaded for each element of the list : I guess it needs batching

Reply:
resolved for cards by adding "avatar" field in graphql